db_service/dao.go

```diff
@@ -13,19 +13,19 @@
 // limitations under the License.
```

Comment:
This file is auto-generated. You'll see some differences in pk -> id because ids are now treated the same as the multi-column keys.